manual/de/ch_Tools_Assistants.xml

```diff
@@ -5324,15 +5333,65 @@
           </glossentry>
         </glosslist>
       </para>
+
+      <para>Der Vollständigkeit halber sind hier noch einige weitere Kursarten genannt, welche zwar
+        durch &app-fq; ermittelt, jedoch in &app; nicht weiter verwendet werden, da diese in
+        erster Linie der <ulink url="&url-wp-de;Technische_Analyse">Chart-Analyse</ulink> dienen
+        und damit den Rahmen von GnuCash sprengen.
+        <footnote>
+          <para>Den vollständigen Umfang, der von &app-fq; bereitgestellten Informationen, können Sie in
+          mit dem Kommando <command>man &app-fq;</command> unter dem Abschnitt 
```

Comment:
command ist eher als Teil von [cmdsynopsis](https://tdg.docbook.org/tdg/4.5/cmdsynopsis.html) zu sehen.
Hier wäre userinput wohl richtiger.
Das haben wir aber bisher fast überall falsch gemacht und auch in wiki:#Text conventions dokumentiert . :-(